Firmware/ultralcd.cpp

```diff
@@ -5216,7 +5216,7 @@ static void lcd_main_menu()
         MENU_ITEM_FUNCTION_P(_T(MSG_FILAMENTCHANGE), lcd_colorprint_change);//8
 
     if (!printer_recovering()) {
-        if ( moves_planned() || printer_active()) {
+        if ( moves_planned() || printer_active() || fan_check_error == EFCE_REPORTED) {
```

Comment:
Is this needed? 🤔 `printer_active()` should return true when the print is paused. If the printer is idle (not printing) then disabling fan check should be available in Settings menu.